doc/examples/streamline_tools.py

```diff
@@ -80,10 +80,10 @@
 """
 The first of the tracking utilities we'll cover here is ``target``. This
 function takes a set of streamlines and a region of interest (ROI) and returns
-only those streamlines that pass though the ROI. The ROI should be an array
+only those streamlines that pass through the ROI. The ROI should be an array
 such that the voxels that belong to the ROI are ``True`` and all other voxels
 are ``False`` (this type of binary array is sometimes called a mask). This
-function can also exclude all the streamlines that pass though an ROI by
+function can also exclude all the streamlines that pass through an ROI by
```

Comment:
an ROI: a ROI

Reply:
This is a tricky one. I pronounce this as "ar-oh-eye", spelling out each letter, which means that I would write "an ROI", but I know that others pronounce this like the name "Roy", which would have you with "a ROI". I say po-tay-toh, you say po-tah-to. I think it's fine as is.